File: PHP Word Examples/BookmarksApiDemo/getBookmarkContent.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\BookmarksApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new BookmarksApi($configuration);

$name = "Bookmark.docx";
$bookmark_name = 'Test';
$folder = "input";
$storage = null;
$password = null;
$result = $apiInstance->getBookmarkContent(
    $name,
    $bookmark_name,
    $folder,
    $storage,
    $password
);

File: Python Word Examples/BookmarksApiDemo/getBookmarkContent.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.bookmarks_api.BookmarksApi(configuration)

file_name = 'bookmark.docx'
bookmark_name = 'Test'
extra = {'folder': 'input'}
result = api.get_bookmark_content(file_name, bookmark_name, **extra)